Documentation/kubernetes.md

```diff
@@ -120,7 +120,6 @@ To clean up all the artifacts created by the demo, **first cleanup the resources
 
 ```bash
 kubectl delete -f rook-operator.yaml
-kubectl delete -n rook-system daemonset rook-agent
```

Comment:
how is the agent deleted?

Reply:
ahh ok.. the previous command deleted the rook-system namespace and the agents are deployed there